dbms/src/Functions/h3EdgeLengthM.cpp

```diff
@@ -0,0 +1,82 @@
+#include "config_functions.h"
+#if USE_H3
+#    include <Columns/ColumnsNumber.h>
+#    include <DataTypes/DataTypesNumber.h>
+#    include <Functions/FunctionFactory.h>
+#    include <Functions/IFunction.h>
+#    include <Common/typeid_cast.h>
+#    include <ext/range.h>
+
+
+extern "C" {
+#    ifdef __clang__
+#        pragma clang diagnostic push
+#        pragma clang diagnostic ignored "-Wdocumentation"
```

Comment:
Is it possible to do without ignoring this flag? Actually we already have had documentation comment....

Reply:
I'm actually don't know what it is for, I just copy-pasted it from `geoToH3.cpp` I'll remove it

Reply:
In this case probably we can't do without it